src/exercise/02.html

```diff
@@ -6,24 +6,16 @@
 
   <!-- 🐨 add React and ReactDOM on the page.
        💰 Here are script tags that'll do the job:
+  -->
   <script src="https://unpkg.com/react@17.0.0/umd/react.development.js"></script>
   <script src="https://unpkg.com/react-dom@17.0.0/umd/react-dom.development.js"></script>
-  -->
-
   <script type="module">
-    const rootElement = document.getElementById('root')
-
-    // You're going to re-implement this code using React!
-    // 💣 So go ahead and delete this implementation (or comment it out for now)
-    // These three lines are similar to React.createElement
-    const element = document.createElement('div')
-    element.textContent = 'Hello World' // 💰 in React, you set this with the "children" prop
-    element.className = 'container' // 💰 in React, this is also called the "className" prop
-    // This is similar to ReactDOM.render
-    rootElement.append(element)
-
-    // 🐨 Please re-implement the regular document.createElement code above
-    // with these React API calls
-    // 💰 The example in the markdown file should be a good hint for you.
+      const helloSpan = React.createElement( 'span', { children: 'Hello', key: 1 } );
+      const worldSpan = React.createElement( 'span', { children: 'World', key: 2 } );
+      const container = React.createElement( 'div', {
+          className: 'container',
+          children: [ helloSpan, ' ', worldSpan ]
+      } );
+      ReactDOM.render( container,document.getElementById('root') )
```

Comment:
not blocking, but eslint would flag the spaces next to the parentheses, as well as there being no space after the comma
```js
ReactDOM.render(container, document.getElementById('root'));
```